tests/benchmarks/micro_benchmarks/test_nvbandwidth.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+
+from tests.helper.testcase import BenchmarkTestCase
+from superbench.benchmarks import BenchmarkRegistry, ReturnCode, Platform
+
+
+class TestNvBandwidthBenchmark(BenchmarkTestCase, unittest.TestCase):
+    """Test class for NV Bandwidth benchmark."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Hook method for setting up class fixture before running tests in the class."""
+        super().setUpClass()
+        cls.createMockEnvs(cls)
+        cls.createMockFiles(cls, ['bin/nvbandwidth'])
+
+    def test_nvbandwidth_result_parsing_real_output(self):
+        """Test NV Bandwidth benchmark result parsing."""
+        benchmark_name = 'nvbandwidth'
+        (benchmark_class,
+         predefine_params) = BenchmarkRegistry._BenchmarkRegistry__select_benchmark(benchmark_name, Platform.CUDA)
+        assert (benchmark_class)
+
+        benchmark = benchmark_class(benchmark_name, parameters='')
+
+        # Preprocess and validate command
+        assert benchmark._preprocess()
+
+        # Provided raw output
+        raw_output = """
+        nvbandwidth Version: v0.6
+        Built from Git version:
+
+        CUDA Runtime Version: 12040
+        CUDA Driver Version: 12040
+        Driver Version: 550.54.15
+
+        Device 0: NVIDIA GH200 480GB (00000009:01:00)
+
+        Running host_to_device_memcpy_ce.
+        memcpy CE CPU(row) -> GPU(column) bandwidth (GB/s)
```

Comment:
I'm a little confused on this output example, is this test will automatically run on all gpus or only one gpu? why htod have 2 gpus while dtoh have 3 gpus?

Reply:
It will run on all CPUs and GPUs.